docs/intro.rst

```diff
@@ -305,7 +305,7 @@ present in the PATH.
   * ``tar``
 
   These commands can be made available in the PATH via a UNIX-like environment
-  such as `CygWin`_, `MSYS2`_, Babun, or Gow. If they are not all available in
+  such as `CygWin`_, MSYS2, Babun, or Gow. If they are not all available in
```

Comment:
Chocolatey is mentioned just above here so shouldn't the note about this being a preq be mentioned there and maybe it should be in the list above this at about 305

Reply:
DONE, but differently:

There are two difficulties with your proposal:
* The `choco` comamnd is only needed if these other commands are not in the PATH
* While these other commands can be made available by the user via a UNIX-like environment, that is not the case for the `choco` command.

Therefore, I don't think we should simply add the `choco` command to the list above.

Instead, I have tried addressing this by changing the paragraph to:

> These commands can either be made available via a UNIX-like environment such as `CygWin`_, MSYS2, Babun, or Gow, or if they are not available in the PATH, they will be installed using the `Chocolatey package manager`_ by invoking its ``choco`` command.